.pre-commit-config.yaml

```diff
@@ -14,4 +14,4 @@ repos:
     rev: v4.4.0
     hooks:
       - id: check-json
-      - id: check-yaml
+      - id: check-yaml
```

Comment:
Needs a trailing newline (i. e. undo this change).

Reply:
Done